jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/forumAdministration.js

```diff
@@ -634,4 +653,13 @@ function showGroupCreateDialog(event) {
             }
         });
     }
+
+    /**
+     *  When groupManagementDialog in 'edit group' mode this method fills up
+     *  the fields in this dialog.
+     */
+    function fillGroupManagementDialogFields() {
```

Comment:
Can you instead pass the values as the parmaterers? Instead of relying on global state and the order of execution.